runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/HeroSection/HeroBackground/HeroBackground.module.scss

```diff
@@ -8,25 +8,25 @@ $ARROWS_HEIGHT: 176px;
 }
 
 .root {
-    @include gradientBackground(4vw, 12vw, 35vw);
+    @include gradientBackground(4vw, 11vw, 35vw);
     margin-bottom: -20vh;
-    height: 120vh;
+    height: 800px;
     isolation: isolate; 
 }
 
 .heroContent {
     display: flex;
     align-items: flex-start;
     flex-direction: column;
-    justify-content: flex-end;
-    height: 100vh;
+    justify-content: space-between;
+    height: 730px;
 }
 
 .heroHandWrapper {
     position: absolute;
     z-index: -1;
     width: 100%;
-    height: 120vh;
+    height: 1000px;
```

Comment:
if you have const value move it to the top of the file to const var e.g.:
HAND_WRAPPER_HEIGHT: 1000px;